oneflow/core/functional/impl/array_functor.cpp

```diff
@@ -908,13 +910,15 @@ class ReshapeFunctor {
       CHECK_EQ_OR_RETURN(shape.Count(0), x_count)
           << "\n Shape " << shape.ToString() << " is invalid for input shape "
           << x->shape()->ToString();
+      if (*(x->shape()) == shape) return x;
```

Comment:
我们代码的习惯是大括号不省略，其他地方也注意加一下
```suggestion
      if (*(x->shape()) == shape) {
        return x;
      }
```

Reply:
已更新